src/server/models/Baseline.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 const database = require('./database');
-const { TimeInterval } = require('../../common/TimeInterval');
+const { TimeInterval } = import('../../common/TimeInterval.mjs');
```

Comment:
To elaborate a bit more from the extended commit message: Because `TimeInterval.mjs` uses ESModules, we can only import things from the file using ESModule-style imports (not CommonJS).